docs/guides/postgres/quickstart/quickstart.md

```diff
@@ -334,7 +334,7 @@ spec:
   resume: true
 ```
 
-KubeDB operator will notice that `spec.resume` is set to `true`. It will delete the DormantDatabase object and create a new Postgres using `spec.origin` from DormantDatabase.
+KubeDB operator will notice that `spec.resume` is set to `true`. It will delete the DormantDatabase object and create a new PostgreSQL using `spec.origin` from DormantDatabase.
```

Comment:
most of the cases, Postgres refers to CRD Postgres.

Reply:
When it means CRD, we should use `Postgres`. When it means database, `PostgreSQL`

Reply:
Yes. I have used like this. may be somewhere it is missed